bot.py

```diff
@@ -2816,6 +2820,74 @@ async def roll(ctx, die='d6'):
     await ctx.send(f'Rolled a D{str(max_roll)} {str(rolls_to_make)} time(s). You rolled **{str(roll_sum)}** in total!')
 
 
+# TODO: works correctly, but the "keeptrack" variable doesn't output anything yet
+#       (it stores every stage of the sorting as a BIG string with line breaks)
+#       so make it output a text file as an attachment or something idk man
+#       also uncomment the 4 lines in the sort func before you do
+#       also should implement negative numbers
+@bot.command(name='sort', aliases=['moomoosort','oobsort'])
+async def oobsort(ctx, *, arg):
+    """Utility
+    Sort all numbers given after the command (other symbols will be discarded).
+    +sort [numbers]
+    """
+    usage = '+sort [numbers]'
+
+    def bettershuffle(array, start):
+        temparray = []
+        for n in range(0, start):
+            temparray.append(array.pop(0))
```

Comment:
nit: popping from an array (unless it's some fixed distance from the end of the array) is O(n) so it would be more efficient to reverse the input array first
That being said, I am aware that making this Efficient is not exactly the goal here
Just wanted to make sure ur aware. I don't know what level of pain you want here.

Reply:
reversing is also O(n) (specifically would take n/2 operations) so im pretty sure its the same either way and thats fiiiiiiiiine given the Interesting efficiency otherwise

Reply:
yeah reversing is O(n) but you're doing an O(n) operation inside of a loop that seems like it could be O(n) in itself so this is O(n^2)

Reply:
oh yeah i guess so, i'll change that then should be ez